lib/riemann/proto.ex

```diff
@@ -1,12 +1,8 @@
 defmodule Riemann.Proto do
-  # this is broken, "from" isn't executed in the context of this module. :(
-  # @proto_file Path.expand("riemann.proto", __DIR__)
-  # @external_resource @proto_file
-  # use Protobuf, from: @proto_file
-
-  alias Riemann.Helpers
   use Protobuf, from: Path.expand("riemann.proto", __DIR__)
-  use_in "Event", Helpers.Event
-  use_in "Msg",   Helpers.Msg
-  use_in "Attribute", Helpers.Attribute
+  @external_resource Path.expand("riemann.proto", __DIR__)
```

Comment:
I tried setting the result of `Path.expand("riemann.proto", __DIR__)` to a common module variable, and referencing that variable when calling `@external_resource` and `use Protobuf, from: ...`, but Protobuf was giving me compilation issues.